samples/sampler/stories/default/ArcPicker.js

```diff
@@ -0,0 +1,33 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {number, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import ArcPicker, {ArcPickerBase} from '@enact/agate/ArcPicker';
+
+ArcPicker.displayName = 'ArcPicker';
+const Config = mergeComponentMetadata('FanSpeedControl', ArcPicker, ArcPickerBase);
+
+storiesOf('Agate', module)
+	.add(
+		'ArcPicker',
+		() => {
+			const itemCount = number('items', Config, {range: true, min: 0, max: 40}, 8);
+			const items = (new Array(itemCount)).fill().map((i, index) => index + 1);
+
+			return (
+				<ArcPicker
+					endAngle={number('endAngle', Config, {range: true, min: 0, max: 360})}
+					options={items}
```

Comment:
Let's define items as `children` prop like the `Picker` instead of the `options`.
```
				<Picker
					disabled={boolean('disabled', Config)}
					onChange={action('onChange')}
					orientation={select('orientation', ['vertical', 'horizontal'], Config)}
				>
					{['LO', '16\xB0', '17\xB0', '18\xB0', '19\xB0', 'HI']}
				</Picker>
```

Reply:
If you need a internal variable for the children, we could name it to `values` like the `Picker`.

Reply:
Is this really necessary? We already have children for the Icon and the span.